packages/react-components/src/lib/components/NoticeBanner.tsx

```diff
@@ -0,0 +1,90 @@
+import { FiX } from 'react-icons/fi';
+import { GrAnnounce } from 'react-icons/gr';
+import { motion, AnimatePresence } from 'framer-motion';
+import { useState } from 'react';
+import { activeNotices, loadNetworkConfig } from '../utils/networkConfig';
+import { atom, useAtomValue } from 'jotai';
+import { loadable, atomWithStorage } from 'jotai/utils';
+
+const networkConfigs = {
+  mainnet: {
+    label: 'Agoric Mainnet',
+    url: 'https://main.agoric.net/network-config',
+  },
+  testnet: {
+    label: 'Agoric Testnet',
+    url: 'https://testnet.agoric.net/network-config',
+  },
+  devnet: {
+    label: 'Agoric Devnet',
+    url: 'https://devnet.agoric.net/network-config',
+  },
+  ollinet: {
+    label: 'Agoric Ollinet',
+    url: 'https://ollinet.agoric.net/network-config',
+  },
+  emerynet: {
+    label: 'Agoric Emerynet',
+    url: 'https://emerynet.agoric.net/network-config',
+  },
+  localhost: {
+    label: 'Local Network',
+    url: 'https://wallet.agoric.app/wallet/network-config',
+  },
+};
+
+const networkConfigAtom = atomWithStorage(
```

Comment:
I think we should just have the component take the loaded network config JSON as an argument instead of hardcoding and fetching the network configs here. I have some ideas for handling network-configs more holistically in `AgoricProvider`, but I'm still not convinced we want the react components to deal with network configs.